README.md

```diff
@@ -1,4 +1,4 @@
-# <img src="https://raw.githubusercontent.com/bareos/bareos/master/webui/public/img/bareos.png" alt="Bareos" />
+# <img src="https://raw.githubusercontent.com/bareos/bareos/master/webui/public/themes/default/img/bareos.png" alt="Bareos" />
```

Comment:
Currently we show the logo from the sunflower theme. did you change this intentionally? If yes, how about using thew logo from https://github.com/bareos/bareos/raw/master/docs/manuals/images/bareos-full-logo.png ?

Reply:
Well, currently it's shown there because we simply have overwritten the original logos in the referenced webui folder. So the replacement in the README.md wasn't really on purpose back in the days, I think.

Reply:
Going to replace the logo in README.md with the one located in docs.

Reply:
Image source replaced, see https://github.com/bareos/bareos/pull/1281/commits/8970e82ef1270b25872431d73ee08cab33dece63 .

Reply:
I think we can resolve this conversation, do you agree?